### ISSUE TITLE
Toolbar is not keyboard accessible 

### ISSUE THREAD

**Issue by ghost** [tinymce/tinymce-react#210]
I am using tinymce latest version i.e 3.9.0.  I cannot access toolbar(with Undo, Redo, Bold, Italic Icons) using Tab key or Alt+F9 key.
Also, in this story book example [Storybook Example Link](https://tinymce.github.io/tinymce-react/?path=/story/tinymce-react--disable-button), If I enable tinymce using enable button on top. And try to access toolbar, the sequence of focus is as follows
1. Disable button is focused
2. Text Area of the Editor is focused(the toolbars are skipped)
Also I am not able to access toolbar using Alt+F9 (as written in documentation), How to make the toolbar keyboard accessible?

**Comment by tiny-james** [comment]
The toolbar buttons should be accessed with ALT+F10, ALT-F9 will access the menus. It is expected that they can not be accessed by the tab key on its own. I checked on Window that I could access them as expected.

I know on some Linux operating systems (sadly including the one I run) those keyboard shortcuts are intercepted by the operating system and causes resizing of the window - is this the problem you are running into?

**Comment by tiny-james** [comment]
Note keyboard shortcuts are here:
https://www.tiny.cloud/docs/advanced/keyboard-shortcuts/

**Comment by tiny-james** [comment]
I'm closing this issue because I am unable to replicate it and I'm probably unable to fix it on the react integration as all accessibility issues are handled by TinyMCE itself.

If you believe there is still a bug please open an issue on the [TinyMCE repository](https://github.com/tinymce/tinymce) and include the browser and operating system you used to replicate the bug - as keyboard shortcuts are OS dependent and browsers do behave very differently with these sorts of issues.
